README.rst

```diff
@@ -19,33 +19,75 @@ To install from PyPi
 
     $ pip install signalfx_lambda
 
+Auto-Instrumentation
+~~~~~~~~~~~~~~~~~~~~
+
+If the Python auto-instrumentation libraries are available to the wrapper, the
+Lambda handler will be auto-instrumented.
+
+To install the supported instrumentation libraries:
+
+::
+
+    $ pip install signalfx-tracing
+
+    $ sfx-py-trace-bootstrap -t <deployment_package_path>
+
+For more information, check out the full :new-page:`instructions <https://github.com/signalfx/signalfx-python-tracing#installation-and-configuration>`.
+
 Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-    SIGNALFX_AUTH_TOKEN=access token
+    SIGNALFX_ACCESS_TOKEN=access token
+
+    # endpoint for both metrics and tracer. Overridden by SIGNALFX_METRICS_URL
+    # and SIGNALFX_TRACING_URL if set
+    SIGNALFX_ENDPOINT_URL=endpoint url
 
-    # optional
+    # optional metrics and tracing configuration
 
-    SIGNALFX_INGEST_ENDPOINT=ingest endpoint [ default: https://pops.signalfx.com ]
+    SIGNALFX_METRICS_URL=ingest endpoint [ default: https://pops.signalfx.com ]
     SIGNALFX_SEND_TIMEOUT=timeout in seconds for sending datapoint [ default: 0.3 ]
 
+    SIGNALFX_TRACING_URL=tracing endpoint [ default: https://ingest.signalfx.com/v1/trace ]
+
+SIGNALFX_TRACING_URL can be used to configure a common endpoint for metrics and
+traces, as is the case when forwarding with the Smart Gateway.
+
+If either SIGNALFX_TRACING_URL or SIGNALFX_METRICS_URL are set, they will take
+precendence over SIGNALFX_TRACING_URL for their respective components.
+
```

Comment:
I'm having a hard time understanding the precedence here, likely because the final usage of `SIGNALFX_TRACING_URL` that's a likely typo?  Would you mind providing a bit more clarification and examples using a gateway and pops (if that makes sense)?  Also, removing `SIGNALFX_INGEST_ENDPOINT` seems to be a breaking change so bw-compatibility would be desired afaict.

Reply:
Fixed the typo, thanks. Also added an example.

I agree that backwards-compatibility would be ideal, but keeping SIGNALFX_INGEST_ENDPOINT as an alias for SIGNALFX_METRICS_URL seems like it would add the potential for confusion in the documentation.

Another option would be to allow both, but only for compatibility, and we don't document SIGNALFX_INGEST_ENDPOINT so we can push new users to use the more descriptive environment variables but not break existing users who upgrade. That doesn't seem great either.

My preference is to make a clean transition to the new names, since I'd imagine the only people upgrading would be the ones who want tracing and would have to add configuration anyway. But I would like to tap into the collective wisdom here on the best approach for this.

Reply:
Fixed with DeprecationWarning